forge.config.js

```diff
@@ -204,37 +223,37 @@ const plugins = [
 		[FuseV1Options.EnableNodeOptionsEnvironmentVariable]: false,
 		[FuseV1Options.EnableNodeCliInspectArguments]: false,
 		[FuseV1Options.EnableEmbeddedAsarIntegrityValidation]: true,
-		[FuseV1Options.OnlyLoadAppFromAsar]: ASAR_ENABLED,
+		[FuseV1Options.OnlyLoadAppFromAsar]: ASAR,
 	}),
 ]
 
-if (ASAR_ENABLED) {
+if (ASAR) {
 	plugins.push(new AutoUnpackNativesPlugin({}))
 }
 
 /** @type {ForgeConfig} */
 export default {
 	packagerConfig: {
-		asar: ASAR_ENABLED,
-		name: 'CoMapeo Desktop',
+		asar: ASAR,
+		name: `CoMapeo Desktop`,
 		icon: './assets/icon',
 	},
 	rebuildConfig: {},
 	makers: [
-		new MakerSquirrel({
-			setupIcon: './assets/icon.ico',
-		}),
-		new MakerDMG({
-			icon: './assets/icon.icns',
-		}),
+		new MakerSquirrel({ setupIcon: './assets/icon.ico' }),
+		new MakerDMG({ icon: './assets/icon.icns' }),
 		new MakerZIP(undefined, ['darwin']),
-		new MakerDeb(
-			{
-				options: { icon: './assets/icon.png' },
-			},
-			['linux'],
-		),
+		new MakerDeb({ options: { icon: './assets/icon.png' } }, ['linux']),
 		new MakerRpm({ options: { icon: './assets/icon.png' } }, ['linux']),
 	],
+	publishers: [
+		new PublisherGithub({
+			draft: true,
+			force: true,
+			generateReleaseNotes: true,
+			repository: { owner: 'digidem', name: 'comapeo-desktop' },
+			tagPrefix: 'v',
+		}),
```

Comment:
as noted in the PR description, we have this set up to create release drafts that:

1. require human intervention to formally publish
2. generates a changelog in the release body text.